src/main/java/com/GeorgesServer/app/com/GeorgesServer/handler/DefaultHandler.java

```diff
@@ -2,20 +2,41 @@
 
 
 import com.GeorgesServer.app.com.GeorgesServer.request.ClientRequest;
-import com.GeorgesServer.app.com.GeorgesServer.response.HttpResponseBuilder;
-import com.GeorgesServer.app.com.GeorgesServer.response.ServerResponse;
+
+import java.util.HashMap;
 
 public class DefaultHandler implements IHandler {
+    private String body;
+    private String status;
+    private HashMap<String,String> headers;
+
+    public void handle(ClientRequest clientRequest) {
+        setStatus();
+    }
+
+    public void setStatus() {
+        this.status = "HTTP/1.1 200 OK";
```

Comment:
This might be a sensible default given your use cases, but defaulting to 'everythings fine' can have a downside.